platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/BirthRoundStateMigration.java

```diff
@@ -46,11 +47,13 @@ private BirthRoundStateMigration() {}
      * @param initialState the initial state the platform is starting with
      * @param ancientMode  the current ancient mode
      * @param appVersion   the current application version
+     * @param platformConfiguration the platform configuration
      */
     public static void modifyStateForBirthRoundMigration(
             @NonNull final SignedState initialState,
             @NonNull final AncientMode ancientMode,
-            @NonNull final SoftwareVersion appVersion) {
+            @NonNull final SoftwareVersion appVersion,
+            @NonNull final Configuration platformConfiguration) {
```

Comment:
Why is this being added if it's not even used?

Reply:
Removed.